src/darwin/Framework/CHIP/MTRDeviceControllerParameters.h

```diff
@@ -79,8 +79,10 @@ MTR_AVAILABLE(ios(17.6), macos(14.6), watchos(10.6), tvos(17.6))
 
 /**
  * Sets the maximum subscriptions allowed for devices on Thread. This defaults to 3.
+ *
+ * If this value is 0, the maximum subscriptinos allowed will be set to 1.
```

Comment:
```suggestion
 * If this value is 0, the maximum subscriptions allowed will be set to 1.
```
But again: I feel like we should have an explicit way to signal "use your default" or "use your best guess" or whatever...

Reply:
When the parameters object is created, this value is set to the default value (of 3).